{"uuid":"9fd6e77a-4914-4989-a066-1379e1f2121c","historyId":"8f03563ea964475080b425beaa0be48d","fullName":"com.dsalgo.qa.testcases.MainPageAndLoginTest.successfulLogin","labels":[{"name":"package","value":"com.dsalgo.qa.testcases.MainPageAndLoginTest"},{"name":"testClass","value":"com.dsalgo.qa.testcases.MainPageAndLoginTest"},{"name":"testMethod","value":"successfulLogin"},{"name":"parentSuite","value":"dsalgoProject"},{"name":"suite","value":"dsalgoRegression"},{"name":"subSuite","value":"com.dsalgo.qa.testcases.MainPageAndLoginTest"},{"name":"host","value":"DESKTOP-US9O7O0"},{"name":"thread","value":"19608@DESKTOP-US9O7O0.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Story Name: To check the successful login scenario"},{"name":"severity","value":"blocker"}],"links":[],"name":"successfulLogin","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Numpy Ninja5] but found [Numpy Ninja]","trace":"java.lang.AssertionError: expected [Numpy Ninja5] but found [Numpy Ninja]\r\n\tat org.testng.Assert.fail(Assert.java:97)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:136)\r\n\tat org.testng.Assert.assertEquals(Assert.java:118)\r\n\tat org.testng.Assert.assertEquals(Assert.java:575)\r\n\tat org.testng.Assert.assertEquals(Assert.java:585)\r\n\tat com.dsalgo.qa.testcases.MainPageAndLoginTest.successfulLogin(MainPageAndLoginTest.java:86)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.lang.reflect.Method.invoke(Method.java:498)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)\r\n\tat org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.util.ArrayList.forEach(ArrayList.java:1259)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:766)\r\n\tat org.testng.TestRunner.run(TestRunner.java:587)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:384)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:286)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1109)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1039)\r\n\tat org.testng.TestNG.run(TestNG.java:1007)\r\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)\r\n\tat org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)\r\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)\r\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)\r\n"},"stage":"finished","description":"Test Case to verify the user is logged in successfully","steps":[{"name":"Get the Main Page title","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1652724585641,"stop":1652724585659}],"attachments":[{"name":"Page screenshot","source":"f4cbd141-d0a6-4f81-8fe8-a932f9d9703b-attachment","type":"image/png"},{"name":"successfulLogin failed and screenshot taken!","source":"7e20510d-ea60-44c1-8aff-863b67c97032-attachment","type":"text/plain"}],"parameters":[],"start":1652724585640,"stop":1652724588771}